test/README.md

```diff
@@ -0,0 +1,22 @@
+Scylla in-source tests.
+
+For details on how to run the tests, see docs/dev/testing.md
+
+Shared C++ utils, libraries are in lib/, for Pythoin - pylib/
```

Comment:
Python.
Note that the pylib functions are only "shared" by a specific set of Python tests that use a specific coding philosophy. 95% of the Python tests we have today don't use them.

Reply:
I fixed the typo. Well, pylib/ is for everyone to share their libs, if some tests don't use it it doesn't mean it shouldn't be mentioned.